yarn.lock

```diff
@@ -1949,6 +1949,22 @@
     "@griffel/react" "^1.5.14"
     "@swc/helpers" "^0.5.1"
 
+"@fluentui/react-list-preview@*":
```

Comment:
hmm this is wrong, seems the generator fixes are missing some projects that used the old `-preview` package and didn't replace it with the new stable one